clients/privacy-center/app/server-environment.ts

```diff
@@ -343,9 +343,9 @@ export const loadPrivacyCenterEnvironment =
       IS_FORCED_TCF: process.env.FIDES_PRIVACY_CENTER__IS_FORCED_TCF
         ? process.env.FIDES_PRIVACY_CENTER__IS_FORCED_TCF === "true"
         : false,
-      GPP_EXTENSION_PATH:
-        process.env.FIDES_PRIVACY_CENTER__GPP_EXTENSION_PATH ||
-        "/fides-ext-gpp.js",
+      FIDES_JS_BASE_URL:
+        process.env.FIDES_PRIVACY_CENTER__FIDES_JS_BASE_URL ||
+        "http://localhost:3000",
```

Comment:
this does mean that running on localhost:3001 won't work for GPP unless this variable is also updated for GPP, which might cause some dev mysteries 😞